stream-chat-android-ui-components/src/main/res/layout/stream_ui_activity_attachment_gallery.xml

```diff
@@ -6,86 +6,155 @@
+        android:background="@color/stream_ui_white_snow"
+        app:layout_constraintBottom_toTopOf="@id/bottomBar"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/toolbar"
         />
 
-    <androidx.constraintlayout.widget.Group
-        android:id="@+id/toolBarGroup"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:constraint_referenced_ids="closeButton, title, subtitle, menuButton"
-        />
 
-    <io.getstream.chat.android.ui.gallery.AttachmentGallery
-        android:id="@+id/attachmentGallery"
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/bottomBar"
         android:layout_width="0dp"
-        android:layout_height="0dp"
+        android:layout_height="56dp"
+        android:background="@color/stream_ui_white"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/subtitle"
-        />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        >
+
+        <View
+            android:layout_width="0dp"
+            android:layout_height="@dimen/stream_ui_divider_height"
+            android:background="@drawable/stream_ui_divider"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+
+        <TextView
+            android:id="@+id/imageCountTextView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textAppearance="@style/StreamUiTextAppearance.HeadlineBold"
+            android:textColor="@color/stream_ui_black"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="1 of 4"
+            />
+
+        <ImageView
+            android:id="@+id/shareImageButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/stream_ui_spacing_small"
+            android:background="?selectableItemBackgroundBorderless"
+            android:padding="6dp"
+            android:src="@drawable/stream_ui_ic_share"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:ignore="ContentDescription"
+            />
+
+        <ImageView
+            android:id="@+id/galleryOverviewButton"
```

Comment:
These buttons don't have the correct colour in dark mode

<img src="https://user-images.githubusercontent.com/12054216/106870895-ef110480-66d1-11eb-8b4d-e1bd48f45cf4.png" width="400"/>

Reply:
Done